tests/utils/spatial_lda_utils_test.py

```diff
@@ -51,11 +51,11 @@ def test_check_format_cell_table_args():
         spu.check_format_cell_table_args(valid_df, invalid_markers1, None)
     with pytest.raises(ValueError):
         spu.check_format_cell_table_args(valid_df, invalid_markers2, None)
-    with pytest.raises(ValueError, match=r"List arguments cannot be empty"):
+    with pytest.raises(ValueError, match=r"empty"):
```

Comment:
`alpineer.misc_utils.verify_in_list` will return `True` if the lists are empty, do you think it would be better to do something else? Here is the PR for it: https://github.com/angelolab/alpineer/pull/25

Reply:
We can discuss at pipeline today. I think it really just depends on what we're using `verify_in_lists` for and if empty lists are considered something we would want to know about.

Reply:
We could handle this by setting a flag to override the default set behavior of empty lists. Something like `fail_on_empty` which defaults to `True`. 

Since I'd imagine empty lists aren't the desired behavior for many users, we could also completely change `verify_in_list` and `verify_same_elements` to fail immediately on discovery of an empty list.

Reply:
@srivarra  @alex-l-kong For now, I'm going to check within the function for an empty list. We can make long term changes to alpineer later if needed.